+) subscription

bbaronSVK opened the pull request.

E000001 committed:
Merge pull request #10 from bbaronSVK/master

add most popular

Commit:
refactor [Add-to-library] feature, so we don not end up with xmbc* dependencies in provider code

Commit:
re-enable cs/en language support based on kodi/user language

Commit:
version 1.1.0

Commit:
+) subscription

Commit:
o) fix series naming

Commit:
o) update video library after link new content

bbaronSVK commented:
pre spravne fungovanie odporucam zmazat vsetky nalinkovane serialy a znova ich nalinkovat. Nasledne sa updatne video kniznica a pridane serialy sa budu same updatovat raz za hodinu.

Commit:
o) fix update library

Commit:
+) mark subscription

lzoubek commented:
@bbaronSVK diky za dalsi cool featuru! Muzes si prosim git rebase master? trosku jsem tu tvou feature prepsal - hlavni co jsem udelal, byl presun xbmc zavislosti z sosac.py do default.py. Zvlast u tohoto pluginu mam zajem na tom, aby bez problemu slapal mimo kodi

Muzes taky prosim udelat pravidelnost scanu konfigurovatelnou? Jednou za hodinu me prijde moc casto (potom se nemuzem divit, ze web sosac kolikrat nestiha)

Commit:
o) prekopane

Commit:
Merge branch 'master' of https://github.com/bbaronSVK/plugin.video.sosac.ph

Conflicts:
	addon.xml
	resources/lib/sosac.py
	resources/settings.xml

bbaronSVK commented:
@lzoubek pozri sa, ci je to ok tak ako som to prepisal. Teraz sa uz pouziva sqlite pre ukladanie subscription. opravil som aj problemy s vytvaranim suborov s diakritikou atd...

lzoubek commented:
mrknu na to vecer, kdyz jsem se dival minule, pull request furt neslo mergnout, ted uz to jde

lzoubek commented on resources/lib/sosac.py:
```diff
@@ -20,14 +20,14 @@
 # */
 
-import re,os,urllib,urllib2,cookielib
+import re,os,urllib,urllib2,cookielib,json,xbmcaddon
```
muzes odstranit zavislost na xbmcaddon?

bbaronSVK replied:
odstranim to ak to tam nikde nepouzivam. json som predtym pouzival ale uz nepouzivam....

lzoubek commented on resources/lib/sutils.py:
```diff
@@ -0,0 +1,187 @@
+        xbmcprovider.XBMCMultiResolverContentProvider.__init__(self,provider,settings,addon)
+        provider.parent = self
+        try: 
+            import sqlite3
```
Nepremyslel jsi spis o pouziti pluginu/sluzby script.common.plugin.cache ? Maji rozumne API pro ukladani dat a pouzivaji taky sqlite, navic resi i zamykani ...

bbaronSVK replied:
nerozmyslal, lebo som ani nevedel, ze nieco take existuje. Pozriem to.

bbaronSVK commented:
Tak som to pomenil...pouzil som script.common.plugin.cache
@lzoubek pozris to pls?

Commit:
o) zmena nacitanych dat
o) zmena generovania nfo suboru pre cely serial, nie pre kazdu epizodu
-) spustenie subscription hned po spusteni

lzoubek commented:
Pridani filmu do knihovny nefunguje, navic se nezobrazi zadna chyba 

`19:24:55 T:139732392519424  NOTICE: ["Initializing provider sosac.ph with settings {'downloads': '', 'quality': '0', 'subs': True}"]
19:24:55 T:139732392519424  NOTICE: ('PARAMS: ', {'url': 'http://movies.prehraj.me/cs/player/a-byla-z-toho-laska-2008', 'action': 'add-to-library', 'name': 'A byla z toho l\xc3\xa1ska (2008)'})
19:24:55 T:139732392519424  NOTICE: ('path: ', 'special://profile/addon_data/plugin.video.sosac.ph/Movies/A byla z toho laska (2008)/A byla z toho laska (2008).strm')
19:24:55 T:139732392519424   ERROR: Create - Error creating special:/profile
19:24:55 T:139732392519424   ERROR: Create - Error creating special:/profile/addon_data
19:24:55 T:139732392519424   ERROR: Create - Error creating special:/profile/addon_data/plugin.video.sosac.ph
19:24:55 T:139732392519424   ERROR: Create - Error creating special:/profile/addon_data/plugin.video.sosac.ph/Movies
19:24:55 T:139732392519424   ERROR: Create - Error creating special:/profile/addon_data`

sonics007 commented:
Libor mne tofunguje, po kliknuti treba dat vyhledat novy obsah v zdroji kde ti uklada filmy

lzoubek commented:
problem je v tom, ze vychozi nastaveni slozky pro filmy/serialy je `special://profile/addon_data/plugin.video.sosac.ph` a tu plugin neumi vytvorit a proto se zadne strm soubory neulozi. Opravit se to da  myslim pomoci `xbmc.translatePath``

sonics007 commented:
No mne to funguje s vychodzim nastavenim ci na andoide ci na win.je to zaujimave. Nemozes to fixnut a   aktualizovat doplnok?

bbaronSVK commented:
@lzoubek tak som to opravil aj s default hodnotami, neviem ako mi ten translatePath vypadol...

Commit:
o) fix s adresarom
+) pridane resolvovanie nazvu serialu z thetvdb fallback z bbdb a nasledne znova z thetvdb

lzoubek commented:
diky, vyzkousim

lzoubek commented:
k cemu je dobry soubor v pluginu resource/.DS_Store?

bbaronSVK commented:
To mam z macu ak som to tam daval ja...ale nepamatam si ze by som to commitoval

bbaronSVK commented:
_resource/.DS_Store_ tam je od zaciatku, ja som z toho vonku. V kazdom pripade to je asi z OSX

Commit:
+) vsetky filmy do kniznice :D z menu

lzoubek commented:
tak to nebude ode me :-) smaznu ho .. diky

Commit:
o) pridat vsetky filmy/serialy do kniznice

Commit:
o) dalsie zmeny upravy

Commit:
o) fix s resolvovanim nazvu

Commit:
o) moznost odstranit serial zo subscription ale nie z kniznice

Commit:
o) upravene zavislosti

bbaronSVK commented:
@lzoubek ako to vyzera?

lzoubek commented:
@bbaronSVK jeste jsem se k tomu nedostal, dnes na to kouknu a uz to doufam mergnu :)

tomtom02 commented:
Ahoj @bbaronSVK  neviem preco mi hadze 'connection error' sc plugin na mojich android boxoch, ci nvidiiTV - ked som v UK, 
ale na slovensku to funguje. 

Vedel by si mi poradit, ako toto vyriesit ked som v UK, prosim pozriet debug, ze v com je chyba? 
nefunkcny debug v UK: https://paste.kodi.tv/ipiwupexuk
vs funkcny kontrolny zo slovenska: https://paste.kodi.tv/ruvisiquti
